src/node/node_state.h

```diff
@@ -972,6 +972,20 @@ namespace ccf
       g.create_service(network.identity->cert);
       g.retire_active_nodes();
 
+      if (network.consensus_type == ConsensusType::BFT)
+      {
+        // BFT consensus requires a stable order of node IDs so that the
+        // primary node in a given view can be computed deterministically by
+        // all nodes in the network
+        // See https://github.com/microsoft/CCF/issues/1852
```

Comment:
@ashamis so as discussed, we will unify ids with CFT, but keep a mapping of node_id -> election_order_id and use that to decide election order, right? If that's right, can it be recorded in #1852.